docs/serviceregistry/manuals/temperature-http-demo.md

```diff
@@ -0,0 +1,846 @@
+  -d '{
+  "instanceIds": [
+  ],
+  "providerNames": [
+    "temperature-provider2"
+  ],
+  "serviceDefinitionNames": [
+    "alert-service"
+  ],
+  "versions": [
+    "1.0.0"
+  ],
+  "alivesAt": "",
+  "metadataRequirementsList": [
+  ],
+  "interfaceTemplateNames": [
+  ],
+  "interfacePropertyRequirementsList": [
+  ],
+  "policies": [
+  ]
+}'
+~~~
+
+The lookup response:
+
+~~~
+{
+  "entries": [
+    {
+      "instanceId": "temperature-provider2::alert-service::1.0.0",
+      "provider": {
+        "name": "temperature-provider2",
+        "metadata": {
+          "scales": [
+            "Kelvin",
+            "Celsius"
+          ],
+          "location": {
+            "side": "North",
+            "block": 2
+          },
+          "indoor": true
+        },
+        "version": "1.0.0",
+        "createdAt": "2024-11-08T10:21:11Z",
+        "updatedAt": "2024-11-08T10:21:11Z"
+      },
+      "serviceDefinition": {
+        "name": "alert-service",
+        "createdAt": "2024-11-08T15:23:10Z",
+        "updatedAt": "2024-11-08T15:23:10Z"
+      },
+      "version": "1.0.0",
+      "expiresAt": "2025-01-01T00:00:00Z",
+      "metadata": {
+        "max-delay": {
+          "value": 15,
+          "unit": "sec"
+        }
+      },
+      "interfaces": [
+        {
+          "templateName": "generic-http",
+          "protocol": "http",
+          "policy": "NONE",
+          "properties": {
+            "accessAddresses": [
+              "192.168.56.116",
+              "tp2.greenhouse.com"
+            ],
+            "accessPort": 8080,
+            "operations": {
+              "subscribe": {
+                "path": "/subscribe",
+                "method": "GET"
+              },
+              "unsubscribe": {
+                "path": "/unsubscribe",
+                "method": "GET"
+              },
+              "set-threshold": {
+                "path": "/threshold",
+                "method": "POST"
+              }
+            },
+            "basePath": "/alert"
+          }
+        }
+      ],
+      "createdAt": "2024-11-10T14:29:13Z",
+      "updatedAt": "2024-11-10T14:29:13Z"
+    }
+  ],
+  "count": 1
+}
+~~~
+
+Since there was exactly one response, we learned that the service instance id is _temperature-provider2::alert-service::1.0.0_. We can revoke the service by sending the following request:
+
+~~~
+curl -X 'DELETE' \
+  'http://localhost:8443/serviceregistry/service-discovery/revoke/temperature-provider2%3A%3Aalert-service%3A%3A1.0.0' \
+  -H 'accept: */*' \
+  -H 'Authorization: Bearer SYSTEM//temperature-provider2'
+~~~
+
+This operation only works, if the service we are deleting is associated with our system. Otherwise, the following error message will be received:
+~~~
+{
+  "errorMessage": "Revoking other systems' service is forbidden",
+  "errorCode": 403,
+  "exceptionType": "FORBIDDEN",
+  "origin": "DELETE /serviceregistry/service-discovery/revoke/{instanceId}"
```

Comment:
Looks like you catch a minor bug. The origin (in case of using path parameters) should contain the actual value of the instanceId, not the literal {instanceId}.

Please fix the demo with the actual value, and create an issue for the bug.